R/language.R

```diff
@@ -20,8 +20,13 @@
 #' q3 <- quote(x + 1)
 #' is_lang(q3)
 #' is_call(q3)
+#'
+#' # A literal is either a scalar atomic vector or a name:
```

Comment:
Maybe `is_leaf()` indicating that it's a leaf of the call tree? 

Do you also need to include `NULL` here?